doc/man3/SSL_read_early_data.pod

```diff
@@ -186,6 +199,19 @@ In the event that the current maximum early data setting for the server is
 different to that originally specified in a session that a client is resuming
 with then the lower of the two values will apply.
 
+Some server applications may wish to have more control over whether early data
+is accepted or not. For example to mitigate replay risks (see L</REPLAY PROTECTION>
+below). The functions SSL_CTX_set_allow_early_data_cb and
```

Comment:
Maybe also say something about declining early data when the server is heavily loaded (this scenario comes up in draft-ietf-httpbis-replay).

Reply:
Added